server/java/scripts/clean.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+mvn clean
```

Comment:
nit: missing extra line at the end

Reply:
the editorconfig should ideally take care of newlines at the end of the file. have added it manually.

Reply:
Newlines at the end are used for a lot of reasons.
A good read can be found here https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline 🙂